_posts/2019-04-10-aws-dashboard-console-using-arduino.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title: Building AWS Dashboard Console using Arduino
+author: m1kma
+excerpt: Arduino compatible microcontrollers are an easy and fun way to create electronic projects. In this blogpost I will cover details of the Arduino project to create a AWS dashboard console prototype.  
+tags:
+- Arduino
+- Electronics
+- AWS
+---
+The system dashboard plays an important role when a real-time system status monitoring and rapid detection of error situations are required. It presents a general view of the running system including metrics, alarms and historical timeseries data. In addition, the dashboard could present special information such as a deployment pipeline status, which is useful especially for the software developers.
+
+Traditional dashboards are usually displayed on computer or TV screen. Therefore, the dashboard must be displayed on the screen to monitor the system status. Is it really necessary? Is there any other way to monitor the system health? An alternative to solve this problem is to build a separate electronic device that is on the table. This device can display the system status f. ex. by lights. The device is an effective way to present system status events immediately without having to see the display. 
+
+![Dashboard console](/img/aws-dashboard-console-using-arduino/dashboard-console.jpg "Dashboard console")
+
+## The AWS Dashboard Console Project
+
+The aim of this project was to create a prototype of the AWS dashboard console using Arduino compatible board and electronic components. The project was mainly a hobby project, but the idea behind was also to create something that I could use on my daily work.
+
+The console presents AWS alerts and codepipeline status by the LED lights and LCD screen. The console runs independently by polling AWS metrics over the WiFi. The console uses NodeMCU microcontroller with an integrated ESP8266 WiFi chip. Just plug-in the power and go. 
```

Comment:
Voisko laittaa jotkut linkit noihin NodeMCH ja ESP8266 WiFi chip kohtiin? :)

Reply:
plug-in --> plug in